AirCasting/CreateSessionViews/TurnOnBluetoothView.swift

```diff
@@ -13,7 +13,7 @@ struct TurnOnBluetoothView: View {
     
     @State private var isPowerABLinkActive = false
     @EnvironmentObject var bluetoothManager: BluetoothManager
-    
+    @EnvironmentObject var settingsRedirection: SettingsRedirection
```

Comment:
It should not use a concrete class here. It should use the protocol you've created for this purpose. When using a concrete class we're loosing all the flexibility that protocols give us